hazelcast/src/main/java/com/hazelcast/client/cp/internal/CPSubsystemImpl.java

```diff
@@ -148,6 +149,12 @@ public boolean removeGroupAvailabilityListener(UUID id) {
         return context.getListenerService().deregisterListener(id);
     }
 
+    @Nonnull
```

Comment:
I guess this will be fixed in the 5.4 release?

Reply:
@pveentjer yes that' sorted in another PR but it's still unsupported here as it's only in EE.